lib/topic_retriever.rb

```diff
@@ -15,7 +15,7 @@ def retrieve
   private
 
   def invalid_url?
-    !EmbeddableHost.url_allowed?(@embed_url)
+    !EmbeddableHost.url_allowed?(@embed_url.strip)
```

Comment:
Is there any chance embed_url can be nil? do we want a `&.` here?

Reply:
There are checks to ensure that `embed_url` is always present.